server/src/main/java/org/apache/druid/server/audit/SQLAuditManagerConfig.java

```diff
@@ -31,6 +31,9 @@
   @JsonProperty
   private boolean includePayloadAsDimensionInMetric = false;
 
+  @JsonProperty
+  private long maxPayloadSizeBytes = -1;
```

Comment:
Could this property declared as type of  `HumanReadableBytes` ?

Reply:
`HumanReadableBytes` won't work here. I am using the value `-1` as disabled this check / unlimited max payload size. This is the same as config like maxBytesInMemory where setting maxBytesInMemory to -1 disables the check.

Reply:
> Does HumanReadableBytes not work with -1?

`HumanReadableBytes` does not work with negative value from config since I think there's no such scenario that a size could be negative. But it can accept negative value from code. 

Here I think there're some ways:
1. if -1 is allowed in configuration, we could remove that constraint in `HumanReadableBytes`, and apply `HumanReadableBytesRange` annotation to make sure its value is in valid range
2. or, if we leave that constraint, user must delete this configuration to disable the check
3. or, provide an independent flag to enable or disable the check as suggested by @jihoonson 

IMO, 3rd is better.

> I noticed we have no unit tests for HumanReadableBytes. It would be nice to have some 🙂.

`HumanReadableBytes` has been covered by UT in [HumanReadableBytesTest](https://github.com/apache/druid/blob/master/core/src/test/java/org/apache/druid/java/util/common/HumanReadableBytesTest.java) file.

Reply:
@jihoonson  Ah, I was wrong. `HumanReadableBytes` does allow `-1`. For example, CaffeineCacheConfig's sizeInBytes also uses `HumanReadableBytes` and set default to `-1`. Changed this to use `HumanReadableBytes`. However, I kept the default value as `-1` which is use for disabling the check. This is consistent with a few other configs we have in Druid right now. We can discuss on changing all of the configs and standardize disable/enable in a separate PR.

Reply:
@FrankChen021 oh you are right. There is a class for unit tests. Sorry, a filter that excludes all test classes might be on in my intellij when I searched last time :sweat: